02-oop-inheritance/javaScript/ver.0.2/main  v0.2.js

```diff
@@ -0,0 +1,58 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+
+var Movie = function (optParams) {
+  this.title = optParams.title;
+  this.duration = optParams.duration; 
+  this.director = optParams.director;
+  this.observer = new MovieObserver();
+};
+
+Movie.prototype = {
+  constructor : Movie,
+
+  getTitle : function () {
+     console.log(this.title);
+  }, 
+
+  setTitle :function (argTitle) {
```

Comment:
there's no need to use argTitle, just use title. No need to change for this exercise